Gruntfile.js

```diff
@@ -105,6 +105,8 @@ module.exports = function(grunt) {
         hot: true,
         port: 3000,
         https: getDevCerts() || true,
+        // host: 'bs-local.com',
```

Comment:
Is this intentional? If true, can we add a link as a comment as well to where the next person can find more context?

Reply:
This was mistakenly committed. Will remove.